setup.py

```diff
@@ -62,7 +62,7 @@
 
 EXTRAS_REQUIRE = {
     "nncf": ["nncf>=2.11.0"],
-    "openvino": ["nncf>=2.11.0", "openvino==2024.5.0", "openvino-tokenizers==2024.5.0"],
+    "openvino": ["nncf>=2.11.0", "openvino>=2024.4.0", "openvino-tokenizers>=2024.4.0"],
```

Comment:
2024.4.0 or 2024.5.0 is lowerbound should be? I see in Helena's PR 2024.5 https://github.com/huggingface/optimum-intel/pull/1005

but both variants works for me

Reply:
2024.4.1 was not released and is not planned to be as I know, only prelrease package exists after that infra switched for preparing 2024.5

Reply:
I am oik with 2024.5. Closing this PR